test/integration/test-all.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+set -e
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+cd "${DIR}"
+
+for testdir in $(find . -type f -name setup); do
+    ("./test-one.sh" "$(dirname ${testdir})")
```

Comment:
what's the reason for the explicit subshell?

Reply:
No good one! Left from a previous refactor. Will remove.